source/reference/minio-server/settings/notifications/amqp.rst

```diff
@@ -40,11 +40,11 @@ For example, the following commands set two distinct AMQP service endpoints as `
       .. code-block:: shell
          :class: copyable
       
-         set MINIO_NOTIFY_AMQP_ENABLE_PRIMARY="on"
-         set MINIO_NOTIFY_AMQP_URL_PRIMARY="amqp://user:password@amqp-endpoint.example.net:5672"
+         export MINIO_NOTIFY_AMQP_ENABLE_PRIMARY="on"
```

Comment:
Is there a `windows` | `not windows` differentiator for this text?

Reply:
no, we don't produce windows docs for the Minio Server reference docs. They're linux only.